pkg/broker/broker.go

```diff
@@ -655,10 +657,44 @@ func (a AnsibleBroker) Provision(instanceUUID uuid.UUID, req *ProvisionRequest,
 			return nil, err
 		}
 	}
-	// TODO: What data needs to be sent back on a response?
-	// Not clear what dashboardURL means in an AnsibleApp context
-	// operation should be the task id from the work_engine
-	return &ProvisionResponse{Operation: token}, nil
+
+	var response *ProvisionResponse
+
+	dashboardRedirectEnabled := func() bool {
```

Comment:
should we make this it's own helper function below and not a function defined in a function? 

Also do we need the dashboard url for [GetServiceInstance](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#fetching-a-service-instance)

Reply:
>but I'm not sure that there is actually anything calling it?
👍 

>  I prefer inline function defs like this because I'm trying to define what it means to be enabled at a higher level when the conditionals start to get more complex than a simple boolean expression in the conditional. I prefer inline because it's close to the call site.

I personally worry about to many inline functions. I believe that at a certain point they make it harder to follow the function. With that said I this one makes sense if it is all encapsulated in another helper function 👍 from me